dev/attest/src/testTsVersions.ts

```diff
@@ -0,0 +1,28 @@
+import { rmSync, writeFileSync } from "node:fs"
+import { join } from "node:path"
+import { exit } from "node:process"
+import { fromPackageRoot, readPackageJson } from "../../runtime/fs.js"
+import { shell } from "../../runtime/shell.js"
+;(() => {
+    const versions: { [k: string]: string } = {
+        "4.8": "16.0.0"
+    }
+    const filePath = join(fromPackageRoot(), ".npmrc")
+    rmSync(filePath, { force: true })
+    writeFileSync(filePath, "save=false")
```

Comment:
Let's add a comment like:

```ts
// Allow us to install a different version of ts-morph for testing without affecting our package.json
```

Since it would be very easy to forget what this does in like... a month KEKW